lua/indent_blankline/init.lua

```diff
@@ -164,7 +162,14 @@ local refresh = function()
     local max_indent_level = v("indent_blankline_indent_level")
     local max_indent_increase = v("indent_blankline_max_indent_increase")
     local expandtab = vim.bo.expandtab
-    local use_ts_indent = v("indent_blankline_use_treesitter") and ts_status and ts_query.has_indents(vim.bo.filetype)
+    local use_ts_indent = false
+    local _, ts_query
+    local _, ts_indent
+    if v("indent_blankline_use_treesitter") then
+        _, ts_query = pcall(require, "nvim-treesitter.query")
+        _, ts_indent = pcall(require, "nvim-treesitter.indent")
+        use_ts_indent = ts_query.has_indents(vim.bo.filetype)
```

Comment:
```suggestion
    local ts_indent
    if v("indent_blankline_use_treesitter") then
        local ts_query_status, ts_query = pcall(require, "nvim-treesitter.query")
        local ts_indent_status
        ts_indent_status, ts_indent = pcall(require, "nvim-treesitter.indent")
        use_ts_indent = ts_query_status and ts_indent_status and ts_query.has_indents(vim.bo.filetype)
```

`ts_query` can be inside the if block.
And you still need to check the status of the `require`